meilisearch/src/extractors/query_parameters.rs

```diff
@@ -0,0 +1,131 @@
+//! For query parameter extractor documentation, see [`Query`].
+
+use std::marker::PhantomData;
+use std::{fmt, ops};
+
+use actix_http::Payload;
+use actix_utils::future::{err, ok, Ready};
+use actix_web::{FromRequest, HttpRequest};
+use deserr::{DeserializeError, DeserializeFromValue};
+use meilisearch_types::error::{Code, ErrorCode, ResponseError};
+
+/// Extract typed information from the request's query.
+///
+/// To extract typed data from the URL query string, the inner type `T` must implement the
+/// [`DeserializeOwned`] trait.
+///
```

Comment:
The documentation seems to be outdated / incorrect?

Reply:
I removed most of the documentation